hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/store/contracts/utils/TokenAccountUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.services.evm.store.contracts.utils;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.datatypes.Address;
+
+public final class TokenAccountUtils {
```

Comment:
We might consider adding `WorldStateTokenAccount` abstraction (e.g. `HederaEvmWorldStateTokenAccount`) inside the `hedera-evm-api` module and then these fields would be kept as static class fields as they currently are.